### PR TITLE
Added Code42 to generic file retrieval, skipped unavailable playbooks

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/32738
fixes: https://github.com/demisto/etc/issues/33378
fixes: https://github.com/demisto/etc/issues/33379

## Description
* Fixed a bug where `Get the binary file from Carbon Black by its MD5 hash` playbook was not skipped if not available
* Fixed a bug where `Get File Sample From Path - Generic V2` would fail if `Get File Sample From Path - VMware Carbon Black EDR - Live Response API` was not available.
* Added the `Code42 File Download` playbook to the `Get File Sample By Hash - Generic v3` playbook

## Screenshots
image

image

## Minimum version of Demisto
5.0.0

## Does it break backward compatibility?
No

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml [hunk 1]
```diff
@@ -129,23 +130,71 @@ tasks:
...
-          "x": 35,
+          "x": -185,
           "y": 195
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
-    skipunavailable: false
+    skipunavailable: true
```
Comment (yaakovi) [inline Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml:140]: nice

**Comment by idovandijk** [comment]
@EliorKedar since you are working on this too, I can merge this PR and you can start working from this point (will save you some work)

**Comment by idovandijk** [comment]
@EliorKedar mentioning you in case you want to be notified when I merge this :)

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3_README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
-This playbook returns a file sample correlating to a hash in the War Room using the following sub-playbooks:
+'This playbook returns a file sample correlating to a hash in the War Room using the following sub-playbooks:'
```
Comment (yaakovi) [inline Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3_README.md:1]: please fix that in the yml file too
```suggestion
This playbook returns a file sample correlating to a hash in the War Room using the following sub-playbooks:
```

**Review by yaakovi — APPROVED** [review]
looks good

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml [hunk 3]
```diff
@@ -62,14 +63,14 @@ tasks:
...
+    taskid: 5ff18884-2925-4b5a-8a1c-be0b0cfe4ac2
     type: playbook
     task:
-      id: a88a3bd6-0201-4203-8e8b-b3962ed19718
+      id: 5ff18884-2925-4b5a-8a1c-be0b0cfe4ac2
       version: -1
       name: Get File Sample By Hash - Cylance Protect v2
       description: This playbook returns a file sample to the War Room given the file's
-        SHA256 hash, using the Cylance Protect v2 integration.
+        SHA256 hash, using Cylance Protect v2 integration.
```
[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_By_Hash_-_Generic_v3.yml:73]: not mine :)

File: Packs/CommonPlaybooks/ReleaseNotes/1_9_0.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+##### Get File Sample By Hash - Generic v3
+- Added the Code42 File Download as a subplaybook to retrieve files using the Code42 integration.
+- Fixed an issue that caused an error in the playbook when one of the subplaybooks was not available.
+
+##### Get File Sample From Path - Generic v2
```
Comment (yaakovi) [inline Packs/CommonPlaybooks/ReleaseNotes/1_9_0.md:7]:
```suggestion
##### Get File Sample From Path - Generic V2
```

**Comment by idovandijk** [comment]
@yaakovi I remember that I saw some fix to this test this morning by you, any idea why it's failing here? (endpoint enrichment)

**Pull request merged**